### PR TITLE
ci: Windows 11 build cli lib

### PR DESCRIPTION
**Pull request by Bykow**
This edition to the workflow allow to build on windows 11 native machine, assuming the machine is provided with a lot of installed tools. This will be documented.

### CODE REVIEW

File: .github/workflows/ci-nym-vpn-cli.yml [hunk 1]
```diff
@@ -13,21 +13,22 @@ on:
...
+  AGENT_ISSELFHOSTED: 1 # https://github.com/actions/setup-go/issues/432
 
 jobs:
   build:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04-arc, custom-runner-mac-m1, windows-2022 ]
-        target: [ native ]
+        os: [ubuntu-22.04-arc, custom-runner-mac-m1, windows11]
```
Comment (octol) [inline .github/workflows/ci-nym-vpn-cli.yml:23]: Is it possible to tweak the label to `custom-windows-11`?

File: .github/workflows/ci-nym-vpn-cli.yml [hunk 2]
```diff
@@ -100,29 +102,84 @@ jobs:
...
 
       - name: Build wireguard (Android)
         if: contains(matrix.target, 'android')
         run: ./wireguard/libwg/build-android.sh
 
+      - name: Checkout mullvad libs for Windows
+        if: contains(matrix.os, 'windows')
+        uses: actions/checkout@v4
+        with:
+          repository: nymtech/nym-vpn-mullvad-libs
```
Comment (octol) [inline .github/workflows/ci-nym-vpn-cli.yml:140]: I think we need to checkout the `nym_patches` branch

File: .github/workflows/ci-nym-vpn-cli.yml (hunk 1 above)
Comment (Bykow) [reply .github/workflows/ci-nym-vpn-cli.yml:23]: Yes

File: .github/workflows/ci-nym-vpn-cli.yml (hunk 2 above)
Comment (Bykow) [reply .github/workflows/ci-nym-vpn-cli.yml:140]: It works as is, I can change if needed.

**Review by octol — APPROVED** [review]

**Pull request merged**